code/modules/admin/verbs/debug.dm

```diff
@@ -837,3 +837,56 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
 	message_admins("<span class='adminnotice'>[key_name_admin(src)] [global.medals_enabled ? "disabled" : "enabled"] the medal hub lockout.</span>")
 	feedback_add_details("admin_verb","TMH") // If...
 	log_admin("[key_name(src)] [global.medals_enabled ? "disabled" : "enabled"] the medal hub lockout.")
+
+
+/proc/make_types_fancy(var/list/types)

```

Comment:
It looks like this might want to interact with the already existing [fancy types system](https://github.com/ParadiseSS13/Paradise/blob/master/code/__HELPERS/unsorted.dm#L1752).